avm/res/web/serverfarm/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -56,8 +56,8 @@ module testDeployment '../../../main.bicep' = [
       name: '${namePrefix}${serviceShort}001'
       location: enforcedLocation
       skuName: 'P1v3'
-      skuCapacity: 3
-      zoneRedundant: true
+      skuCapacity: 2
+      zoneRedundant: false
```

Comment:
I'd suggest to add to the metadata description a brief reason why this had to be set to false although not aligned with WAF recommedations (despite the test name). That way, this will be automatically added to the README test description, providing clarity on the reasons why we decided to proceed that way

Reply:
Great suggestion @eriqua i was wondering where i could mention this. All done.